src/Documentation/resources/BestPractices

```diff
@@ -0,0 +1,422 @@
+# Best Practices
```

Comment:
I think this file should be called best_practices.md, or at least end in .md

Reply:
Good call!

Reply:
done.